src/app.vue

```diff
@@ -18,5 +18,8 @@ import Header from '@/components/header/header-component.vue';
     Header,
   },
 })
-export default class App extends Vue {}
+export default class App extends Vue {
+  // TODO: Set to true when authentication is completed
```

Comment:
Would you mind taking this a bit further and adding an `authentication.service` so that later we can just add the appropriate API request to it?

Reply:
I'm gonna punt this, we will be setting up Authentication soon enough.